examples/http.rs

```diff
@@ -0,0 +1,116 @@
+use std::collections::HashMap;
+
+use anyhow::Result;
+use hyper::header::CONTENT_TYPE;
+use serde::{Deserialize, Serialize};
+
+use esp_idf_svc::hal::prelude::Peripherals;
+use esp_idf_svc::log::EspLogger;
+use esp_idf_svc::wifi::{BlockingWifi, EspWifi};
+use esp_idf_svc::{eventloop::EspSystemEventLoop, nvs::EspDefaultNvsPartition};
+
+use lynx_embedded::{reqwesp, wifi as espWifi};
+
+#[allow(dead_code)]
+#[derive(Deserialize, Debug)]
+struct Anything {
+    pub data: String,
+    pub json: Option<Account>,
+    pub method: String,
+    pub origin: String,
+    pub url: String,
+}
+
+#[derive(Serialize, Deserialize, Debug)]
+struct Account {
+    username: String,
+    password: String,
+}
+
+fn main() -> Result<()> {
+    // Bind the log crate to the ESP Logging facilities
+    EspLogger::initialize_default();
+
+    // Configure Wifi
+    let peripherals = Peripherals::take().unwrap();
+    let sys_loop = EspSystemEventLoop::take()?;
+    let nvs = EspDefaultNvsPartition::take()?;
+
+    let mut wifi = BlockingWifi::wrap(
+        EspWifi::new(peripherals.modem, sys_loop.clone(), Some(nvs))?,
+        sys_loop,
+    )?;
+
+    espWifi::connect(&mut wifi)?;
+
+    let mut client = reqwesp::Client::new()?;
+    let url = "https://httpbin.org/anything";
```

Comment:
Can we add a comment on this url. "Endpoint for testing REST requests." A bit more detail ideally.

Reply:
i mean its a tester/sample file?

Reply:
Yes, this is an example file for demonstrating the usage of the reqwesp module. The front page of https://httpbin.org/ explains the purpose of the site as well.

Reply:
I think a comment is still helpful